### PR TITLE
Remove outdated variables from requestData.h

### PR DESCRIPTION
**Pull request by will889889**
Now we don't pass IP and port to RequestData.
The operation of sending message via socket is belong to Worker.

// this commit also used clang format

### CODE REVIEW

**Comment by john0312** [comment]
Please rebase and split off clang-format changes.

**Comment by will889889** [comment]
There're two commits now, one is for clang format, another is for removing outdated variables of RequestData.

**Review by john0312 — APPROVED** [review]

File: src/RequestData.h [hunk 1]
```diff
@@ -1,40 +1,35 @@
...
+#include <string>
 
-namespace shortlink
-{
-	
-// RequestData is a class of a data structure that represents a request that a user sends to the server.
-// Example:
+namespace shortlink {
+
+enum RequestType { kInvalid, kSearch, kCreate };
```
Comment (john0312) [inline src/RequestData.h:13]: Please add comment for the enum.

**Review by john0312 — COMMENTED** [review]
Please fix comments before merging.

**Pull request merged**